examples/running-tests-using-travis-and-saucelabs/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "my-project",
```

Comment:
Pick some meaningful name for this project like `running-tests-using-travis-and-saucelabs`. Also, it's better to run `npm init` on the repo and fill missing fields